docs/DeveloperGuide.md

```diff
@@ -1059,16 +1054,16 @@ enabling users to identify the problem immediately and take appropriate action.
 **Current**: Our application now imposes limitations on tag length, restricting users from inputting longer tags,
 which may hinder organization and labeling efforts.
 
-**Future**: To address this, we aim to extend the maximum length of tags supported within the system. This enhancement
-will empower users to provide more descriptive labels and organize content more effectively, enhancing usability.
+**Future**: To address this, we aim to increase the maximum length of tags supported within the system, enabling users
+to provide more descriptive labels and organize content effectively.
 
 4. Resolution Support
 
 **Current**: Our application currently only support a single type of screen resolution, limiting compatibility
 with various devices and user preferences.
 
 **Future**: Recognizing the importance of catering to diverse user needs, we plan to expand our resolution support
-to include additional screen resolutions such as 1280x720. This will ensure compatibility across a broader range of
+to support default screen resolution such as 1280x720. This will ensure compatibility across a broader range of
 devices, enhancing accessibility and user experience.
```

Comment:
Our application is currently only optimised for the recommended screen resolution and screen scale for your device. This may limit compatibility with different user preferences. It is usable but not optimal for a small margin of error from your recommended screen resolutions and scale.

**Future**: We plan to expand our support for various screen resolutions and screen scales by including custom scaling, which which will enhance accessibility and improve the user experience.

Reply:
Thanks for the edit